components/base/sensorcontrolled/movestraight.go

```diff
@@ -43,13 +43,26 @@ func (sb *sensorBase) MoveStraight(
 		}
 		return sb.controlledBase.MoveStraight(ctx, distanceMm, mmPerSec, extra)
 	}
+	if sb.position == nil {
+		sb.logger.CWarn(ctx,
+			"Position reporting sensor not available, controlling using linear velocity only")
```

Comment:
not sure if we should remove the warning, can remove if we think its unnecessary

Reply:
maybe something more like "for increased accuracy add a position reporting sensor" so it's just phrased as a helpful addition rather than something is missing?